test/test_files/tinysnb/agg/distinct_agg.test

```diff
@@ -117,3 +117,7 @@ Bob|Dan
 2000-01-01|Dan
 2021-06-30|Carol
 2021-06-30|Dan
+-LOG DistinctRecursiveRel
+-STATEMENT MATCH (a:person)-[t:knows*1..3]-(b:person) RETURN count(DISTINCT t);
```

Comment:
if we do count(t), do we also get 1036? Not sure if we have duplicated `t` in this case.